standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -687,6 +687,8 @@ public static ConfVars getMetaConf(String name) {
         "hive-metastore/_HOST@EXAMPLE.COM",
         "The service principal for the metastore Thrift server. \n" +
             "The special string _HOST will be replaced automatically with the correct host name."),
+    LOCKLESS_READS_ENABLED("metastore.lockless.reads.enabled", "metastore.lockless.reads.enabled",
```

Comment:
I think there should be a separate config just for the drop partition

Reply:
do you mean there should be one main config (e.g. this one) and additional one for each related operation (dropTable/partition/etc) or no main one and one for each op?